WordPress/WordPressTest/ReaderReblogActionTests.swift

```diff
@@ -77,8 +62,10 @@ class ReblogPresenterTests: ReblogTestCase {
 
     func testPresentEditorForOneSite() {
         // Given
+        BlogBuilder(blogService!.managedObjectContext).with(visible: true).isHostedAtWPcom().withAnAccount().build()
         postService!.draftPostExpectation = expectation(description: "createDraftPost was called")
-        blogService!.blogsForAllAccountsExpectation = expectation(description: "blogsForAllAccounts was called")
+        // TODO: Replace this expectation with other ways to assert the `ReaderReblogPresenter.presentEditor` is called.
+//        blogService!.blogsForAllAccountsExpectation = expectation(description: "blogsForAllAccounts was called")
```

Comment:
Do you plan on following up on this shortly? Otherwise, what would be a good place to track this TODO?

Related: I don't know if we have a good process for reviewing the state of `TODO` and `FIXME` notes in the codebase. Keen to discuss ideas on how to tackled that at some point.

Reply:
This TODO should be resolved once we have a solution for https://github.com/wordpress-mobile/WordPress-iOS/issues/19373. What's the best way to associate this TODO with the issue? Do I just put the issue link in the comments?

Reply:
Yeah. I don't know if we have a neat way but a link in the PR will do. I added it to the issue:

<img width="963" alt="image" src="https://user-images.githubusercontent.com/1218433/193940965-d2cbf9c7-8566-4193-8fce-8beb1f7d5632.png">